docs/markdown/Vala.md

```diff
@@ -89,7 +89,7 @@ For the typelib, use a custom target depending on the library:
 
 ```meson
 g_ir_compiler = find_program('g-ir-compiler')
-custom_target('foo typelib', command: [g_ir_compiler, '--output', '@OUTPUT@', '@INPUT@'],
+custom_target('foo typelib', command: [g_ir_compiler, '--shared-library', foo_lib.full_path(), '--output', '@OUTPUT@', '@INPUT@'],
```

Comment:
Maybe someone expert with `g-ir-compiler` could clarify this, because it states the following in the manpage for `--shared-library`:

> Specifies  the shared library where the symbols in the typelib can be found.  The name of the library should not contain the leading lib prefix nor the ending
              shared library suffix.

But just putting `--shared-library=foo` is not working, possibly due to a missing `LD_LIBRARY_PATH`.